File: C/meanmedian.c
#include <stdio.h>
int main(){
	printf("mean__1\nmedian__2\na variable when mean is given__3\n");
    int wh,entries,sum=0,med,new,fin,sum2=0,il3,ent,mea,arr[0],il;
	scanf("%d\n",&wh);
	switch(wh)
	{
	case 1:
		scanf("enter the number of entries%d\n",&entries);
		for (int i = 0; i < entries; i++){
			scanf("enter the data...%d\n",&il);
			sum+=il;
		}
        printf("the mean is... %f\n",(sum/entries) );
        break;
	case 2:
		scanf("enter the number of entries...%d\n",&med);
		arr[med];
		for (int ko=0;ko<med;ko++){
			scanf("enter data__%d\n",&arr[ko]);
		}
		if(med%2!=0){
			 new=med+1;
			printf("the median is....%d\n",arr[(new/2)+1]);
		}
		else{
			 fin=(arr[med/2])+(arr[(med/2)]);
			printf("the median is...%d\n",fin );
		}
		break;
	case 3:
		scanf("enter the mean--%d\nenter the no. of entries..%d\n",&mea,&ent);
		for (int jh=0;jh<ent;jh++){
			scanf("enter data__%d\n",&il3);
            sum2+=il3;
		}
		float var=(mea*ent)-sum2;
		printf("variable is__%d\n",var );
		break;
	default:
	    printf("invalid choice...\n");
}
}

File: java/meanmedian.java
import java.util.*;
public class meanmedian{
public static void main(String h[]){//working......
Scanner nrt=new Scanner(System.in);//success.....
System.out.println("what do you need to find?\nmean__1\nmedian__2\nfind a variable when mean is given__3");
int n=nrt.nextInt();
int j,k,jl,lj,il;
float med,sum=0;
if(n==1){
System.out.println("enter the no. of entries...");
j=nrt.nextInt();
for (k=0;k<j ;k++ ) {
System.out.println("enter the data..");
il=nrt.nextInt();
sum+=il;
}
System.out.println("the mean is> "+(sum/j));
}
if(n==2){
System.out.println("enter the no. of entries...");
jl=nrt.nextInt();
int arr[]=new int[jl];
for (lj=0;lj<jl ;lj++ ) {
arr[lj]=(nrt.nextInt());
}
if(jl%2==0){
med=(1/2)*(((float)Math.floor(arr.length / 2))/2)+((float)Math.floor(arr.length / 2)/2+1);
System.out.println("median is> "+med);
}
else{
med=((float)Math.floor(arr.length / 2))/2;
System.out.println("median is> "+med);
}
}
else if (n==3){
System.out.println("enter the mean...\nenter the no. of entries...");
float mean=nrt.nextFloat();
int ent=nrt.nextInt(),sum2=0;
for (int a=1;a<ent;a++){
sum2+=(nrt.nextInt());
}
float find=(mean*ent)-sum2;
System.out.println("the variable is> "+find);
}
else{
	System.out.println("invalid choice....");
}
}
}